src/moments.jl

```diff
@@ -63,10 +63,18 @@ function var!(R::AbstractArray, A::RealArray, wv::WeightVec, dim::Int; mean=noth
 end
 
 Base.varm(A::RealArray, wv::WeightVec, M::RealArray, dim::Int) =
-    Base.varm!(Array(Float64, Base.reduced_dims(A, dim)), A, wv, M, dim)
+    if VERSION < v"0.5.1"
```

Comment:
Any chance we can be more specific with the version requirements? (Here and elsewhere in the diff)